packages/website/src/components/typeDetails/TypeInfo.tsx

```diff
@@ -67,7 +67,7 @@ export function TypeInfo({
   onHoverNode,
 }: TypeInfoProps): React.JSX.Element {
   const computed = useMemo(() => {
-    if (!typeChecker || !value) {
+    if (!typeChecker) {
```

Comment:
this is not correct

Reply:
@armano2 according to the types it is - the property `value` is defined as `readonly value: Node` and it's never reassigned in the component body